app/uk/gov/hmrc/servicedependencies/model/MongoLibraryVersion.scala

```diff
@@ -18,12 +18,12 @@ package uk.gov.hmrc.servicedependencies.model
 
 import org.joda.time.DateTime
 import play.api.libs.json.Json
-import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
+import uk.gov.hmrc.mongo.play.json.MongoJodaFormats
```

Comment:
Is it worth using the JavaDatetimeFormats instead?

Reply:
Yeah, makes sense while we're refactoring anyway. Have switch all to java time.